Web/Html,CSS,Javascript/github-profiles/script.js

```diff
@@ -28,6 +28,7 @@ async function getRepos(username) {
 }
 
 function createUserCard(user) {
+    const userURL = user.html_url
```

Comment:
Github user does not have any attribute named "html_url". Its only url and that too won't work here.